querybook/webapp/components/DataDocSchedule/DataDocScheduleForm.tsx

```diff
@@ -176,6 +180,18 @@ export const DataDocScheduleForm: React.FunctionComponent<
                 const notificationField = (
                     <>
                         <FormSectionHeader>Notification</FormSectionHeader>
+                        <div>
+                            Notifications will be sent to the user who created
```

Comment:
Notifications will be sent to the user @jczhong84, and they are also the owner of the scheduled queries.